test/runner.html

```diff
@@ -91,6 +91,9 @@
         require(["./widgets"], function () {
             mocha.run();
         });
+        require(["./utility"], function () {
+            mocha.run();
+        });
     </script>
```

Comment:
is there a quick way we can turn on an off a group then? the way it is now i can quickly comment out the moch.run() for any particular one